docs-src/performance-testing.md

```diff
@@ -0,0 +1,48 @@
+Performance testing is possible thanks to `browsermob-proxy`.
+
+It saves all data from network tab (Google Chrome console) which is generated during the test.
+
+###What needs to be done?
+
+#Get started
+
+1. Download `browsermob-proxy` from `https://github.com/lightbody/browsermob-proxy`
+
+2. Navigate in terminal to the catalog
+
+3. Use following command to start the REST API
+
+```
+./browsermob-proxy -port 8887
```

Comment:
It would be nice to check how to run it on Windows - we do support it :)

Reply:
True, we should check it :P